pkg/engine/engine.go

```diff
@@ -887,8 +887,10 @@ func (e *Engine) notifyResults(ctx context.Context) {
 		// want to include duplicate results with the same decoder type.
 		// Duplicate results with the same decoder type SHOULD have their own entry in the
 		// results list, this would happen if the same secret is found multiple times.
+		// Note: If the source type is postman, we dedupe the results regardless of decoder type.
```

Comment:
There would be a bunch of duplicates representing the same information. So for example if we have built say three keywords from walking the postman response (k1, k2, k3) and we have a valid token (token1) in some auth field then the data chunk that we create would look something like:

```
k1: token1
k2: token1
k3: token1
```

We don't want to report 3 different results since `token1` is leaked in _one_ auth field.